stable/secrets/README.md

```diff
@@ -0,0 +1,40 @@
+# secrets
+
+This chart is used to apply secrets. You can specify any number of secrets and any number of values inside a secret. This chart can come in handy when a public chart does not offer additional secrets. You have to use the following format to specify secrets:
```

Comment:
I think the README needs more about the usecases including an example of how it would be used with the 
‘public chart”.

Reply:
Updated